src/controls/generic-controls.js

```diff
@@ -289,25 +289,31 @@ export default class GenericControls {
     }
 
     const KEY_BINDINGS = {
-      KeyW: 'moveForward',
+      w: 'moveForward',
+      W: 'moveForward',
       ArrowUp: 'moveForward',
```

Comment:
`event.key` produces slightly different values from `event.code` but is better supported cross browsers.